packages/ui/src/components/Chip/__stories__/index.stories.tsx

```diff
@@ -0,0 +1,19 @@
+import type { Meta } from '@storybook/react'
+import { Chip } from '..'
+
+export default {
+  component: Chip,
+  decorators: [
+    StoryComponent => (
+      <div style={{ display: 'flex', flexDirection: 'column' }}>
+        <StoryComponent />
+      </div>
+    ),
+  ],
+  title: 'Components/Badges/Chip',
+} as Meta<typeof Chip>
```

Comment:
you miss Chip.Icon here in the parameters so we can see the prop of that component in storybook